### PR TITLE
Fix, encryption enabled on new forum

### PR DESCRIPTION
**Pull request by Claire1817**
#661

### CODE REVIEW

File: vector/src/main/java/im/vector/app/features/roomdirectory/createroom/CreateRoomViewModel.kt [hunk 1]
```diff
@@ -321,19 +321,15 @@ class CreateRoomViewModel @AssistedInject constructor(
 //                            preset = CreateRoomPreset.PRESET_PRIVATE_CHAT
 //                        }
 //                    }
-                    // Disabling federation
-                    disableFederation = state.disableFederation
-
```
Comment (Florian14) [inline vector/src/main/java/im/vector/app/features/roomdirectory/createroom/CreateRoomViewModel.kt:326]: Please can you restore and comment these lines to stay aligned with Element?

File: vector/src/main/java/im/vector/app/features/roomdirectory/createroom/CreateRoomViewModel.kt [hunk 2]
```diff
@@ -321,19 +321,15 @@ class CreateRoomViewModel @AssistedInject constructor(
...
+//                    val shouldEncrypt = when (state.roomJoinRules) {
+//                        // we ignore the isEncrypted for public room as the switch is hidden in this case
+//                        RoomJoinRules.PUBLIC -> false
+//                        else                 -> state.isEncrypted ?: state.defaultEncrypted[state.roomJoinRules].orFalse()
+//                    }
+//                    if (shouldEncrypt) {
+//                        enableEncryption()
+//                    }
 
                     if (state.roomType == TchapRoomType.EXTERNAL) {
```
Comment (Florian14) [inline vector/src/main/java/im/vector/app/features/roomdirectory/createroom/CreateRoomViewModel.kt:334]: I think we can move the following lines (until the federation config) above the commented bloc to have a single bloc dedicated to tchap custom configuration. Eventually, we can move the tchap config to a dedicated method to facilitate the rebases

File: vector/src/main/java/im/vector/app/features/roomdirectory/createroom/CreateRoomViewModel.kt (hunk 1 above)
Comment (Claire1817) [reply vector/src/main/java/im/vector/app/features/roomdirectory/createroom/CreateRoomViewModel.kt:326]: theses lines are present lines 343. There are not commented, should i do it ?

**Review by Florian14 — APPROVED** [review]
LGTM, thanks for the update

**Pull request merged**